build.sc

```diff
@@ -70,7 +70,7 @@ object Deps {
   val testScala33Version = "3.3.1"
 
   object Scalajs_1 {
-    val scalaJsVersion = "1.15.0"
+    val scalaJsVersion = "1.16.0"
```

Comment:
This version is not forward binary compatible with older releases, so users can't use lower versions downstream, which might be an issue for libraries. `1.15.0` was forward compatible down to `1.13.0`. Can you tell how important it is for user to use or stay compatible to older versions? Should we cross-compile our workers to support older versions?

Reply:
Our worker can link correctly from Scala.js `1.0.0` to future `1` versions.
We check the Scala.js version in the linker to avoid using features that don't exist in older versions.